tests/user.test.js

```diff
@@ -109,7 +109,7 @@ describe('User', () => {
       await api
         .put('/users/1')
```

Comment:
Consider using `patch` instead of `put` for updating a user, as the API seems to be using `patch` for partial updates.